traitsui/key_bindings.py

```diff
@@ -294,3 +260,48 @@ def _match_binding(self, binding, skip=frozenset()):
             for a_binding in self.bindings
             if a_binding not in skip and a_binding.match(binding)
         )
+
+
+class KeyBindingsHandler(ModelView):
+
+    bindings = List(Instance(KeyBinding))
+
+    def key_binding_for(self, binding, key_name):
+        """Returns the current binding for a specified key (if any)."""
+        if key_name != "":
+            for a_binding in self._match_binding(key_name, skip={binding}):
+                return a_binding
+
+        return None
+
+    def _match_binding(self, binding, skip=frozenset()):
+        """Return all KeyBinding instances that match the given binding.
+        """
+        return (
+            a_binding
+            for a_binding in self.bindings
+            if a_binding not in skip and a_binding.match(binding)
+        )
+
+    def _bindings_default(self):
+        bindings = list(set(self.model.root._get_bindings([])))
```

Comment:
This API feels a little awkward. Why make `bindings` a necessary argument for `_get_bindings` in the first place? why not make `bindings=None` the default and make `_get_bindings` handle the case internally?

Reply:
Yep, not fond of it myself (it feels old-school), but it works and I wanted to change as little as possible, given the state of testing on the code.